src/ObjectGraph.es6.js

```diff
@@ -722,8 +749,9 @@ module.exports = facade(ObjectGraph, {
     clone: 1, cloneWithout: 1, capture: 1, removeIds: 1, removePrimitives: 1,
     isType: 1, getType: 1, isFunction: 1, getFunctions: 1, getAllIds: 1,
     getObjectKeys: 1, getKeys: 1, getShortestKey: 1, getAllKeys: 1,
-    getAllKeysMap: 1, toJSON: 1, getPrototype: 1, lookup: 1, getFunctionName: 1,
-    getRoot: 1, getPropertiesIds: 1, lookupMetaData: 1,
+    getAllKeysMap: 1, toJSON: 1, getPrototype: 1, getSupers: 1, lookup: 1,
```

Comment:
This would be better as entry per line. Trying to figure out what changed in this block involved more reading than it should have.

Reply:
Will add follow-up commit.